### PR TITLE
Polygon tool convergence

### PR DESCRIPTION
**Pull request by Asliuyong**

### CODE REVIEW

File: packages/lb-annotation/src/core/toolOperation/polygonOperation.ts [hunk 1]
```diff
@@ -1424,21 +1446,25 @@ class PolygonOperation extends BasicToolOperation {
...
+        if (this.isSingleCropping) {
+          this.crop();
+          return;
+        }
+        break;
 
-      return;
+      case 2:
+        this.isCombined = false;
+        this.isSingleCropping = false;
```
Comment (Glenfiddish) [inline packages/lb-annotation/src/core/toolOperation/polygonOperation.ts:1464]: is isSingleCropping can implement in crop? Single and multiple should be handle in a function.

File: packages/lb-components/src/views/MainView/sidebar/OperationRotate/index.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
+/**
+ * @file Rotate polygon
```
Comment (Glenfiddish) [inline packages/lb-components/src/views/MainView/sidebar/OperationRotate/index.tsx:2]: rotate on point cloud

**Review by Glenfiddish — CHANGES_REQUESTED** [review]
Reviewed

File: packages/lb-components/src/views/MainView/sidebar/OperationRotate/index.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  const rotate = (direction: number) => {
+    if (!toolInstance.selectedPolygon?.isRect) {
+      return;
+    }
+    toolInstance.rotatePolygon(1, direction);
+  };
+
+  const dataList = [
+    {
+      name: '左旋1°(←)',
```
Comment (Kerwin-L) [inline packages/lb-components/src/views/MainView/sidebar/OperationRotate/index.tsx:42]: Please add the shortcut key

File: packages/lb-components/src/views/MainView/sidebar/ToolStyle/index.tsx [hunk 4]
```diff
@@ -147,7 +147,10 @@ const ToolStyle = (props: IProps) => {
       {Object.entries(styleConfig).map((item: any[]) => {
         const key: ToolStyleKey = item[0];
         // 判断是否需要 color 的使用，现在暂时默认不需要
-        if (annotationConfig?.attributeConfigurable === true && key === 'color') {
+        if (
+          (annotationConfig?.attributeConfigurable === true || annotationConfig?.lineColor) &&
```
Comment (Kerwin-L) [inline packages/lb-components/src/views/MainView/sidebar/ToolStyle/index.tsx:151]: Need to judge `lineColor` whether `lineColor` is `ELineColor.MultiColor`

File: packages/lb-components/src/components/customResizeHook/index.tsx [hunk 5]
```diff
@@ -63,6 +63,7 @@ export const ViewportProviderComponent = (props: any) => {
         dispatch(PageForward());
       }
       if (e.keyCode === EKeyCode.R) {
+        e.preventDefault()
```
Comment (Kerwin-L) [inline packages/lb-components/src/components/customResizeHook/index.tsx:66]: If it is `Ctrl + R(Command + R)`， then do not `rotate`

**Pull request merged**